sklearn/linear_model/ridge.py

```diff
@@ -1510,14 +1533,15 @@ class _BaseRidgeCV(LinearModel):
     def __init__(self, alphas=(0.1, 1.0, 10.0),
                  fit_intercept=True, normalize=False, scoring=None,
                  cv=None, gcv_mode=None,
-                 store_cv_values=False):
+                 store_cv_values=False, is_clf=False):
```

Comment:
You should not need to pass add this new parameter to `_BaseRidgeCV`.

instead `_BaseRidgeCV` instances are able to know wheter they are a classifier by using `if is_classifier(self)`


`is_classifier` is in `sklearn.base` (use a relative import though)


The reason is that the classes than inherit from `_BaseRidgeCV` also inherit from `RegressorMixin` or `ClassifierMixin`, and these Mixins have the necessary info for `is_classifier` to work.

Reply:
Thanks a lot, I was not aware of this concept. 
Removed the unnecessary param from `_BaseRidgeCV`.